drivers/Ticker.h

```diff
@@ -100,6 +100,19 @@ class Ticker : public TimerEvent, private NonCopyable<Ticker> {
      *
      *  @param func pointer to the function to be called
      *  @param t the time between calls in micro-seconds
+     *
+     *  @note setting @a t to very small value much less than RTOS tick frequency
+     *  can cause system malfunction or even system hang. By default OS tick frequency
+     *  is 1000Hz (1 tick == 1ms) so we schouldn't set much less then 1000us.
+     *  The lowest value which can be set without program hang is H/W dependent
```

Comment:
This note makes it seem like there is a bug with the some implementations of the ticker which should not be the case. Depending on how much processing is done in the ticker interrupt and current cpu speed the values of `t` which make the program hang could be arbitrarily large. For example, if your interrupt routine takes 11ms to complete, then a `t` of 10,000 will cause a system hang.

Maybe a generic warning like this would be more appropriate?
```
setting @a t to a value shorter that it takes to process the ticker callback will cause the system to hang.
```

Reply:
Yeah, maybe you are right. Cutting a bit on details and maybe quoting the hardware specific danger zone below 500us(?) which should be safe for most platforms.

Reply:
I would say that:
timout > T1(calback exec time) + T2(task switch time)

Reply:
If we set Ticker callback time small enought to fall into endless processing loop there ist still chance to recover from this "hung" state putting some ticker control code inside callback which call `detach` in suitable moment.

So maybe it's good idea to make comment/warrning short and leave programmer to decide what is good for him.